api/base/views.py

```diff
@@ -136,6 +149,112 @@ def get_serializer_context(self):
         })
         return context
 
+
+class LinkedNodesRelationship(JSONAPIBaseView, generics.RetrieveUpdateDestroyAPIView, generics.CreateAPIView):
```

Comment:
No, collections use them too. 

.:Brian J. Geiger
Sent from the phone.

> On Jul 5, 2016, at 10:16 AM, Steven Loria <notifications@github.com> wrote:
> 
> In api/base/views.py:
> 
> > @@ -136,6 +149,112 @@ def get_serializer_context(self):
> >          })
> >          return context
> >  
> > +
> > +class LinkedNodesRelationship(JSONAPIBaseView, generics.RetrieveUpdateDestroyAPIView, generics.CreateAPIView):
> This should probably go in api/nodes/views.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>